QAchat/py/QAchat.ipynb

```diff
@@ -34,7 +34,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%pip install -U braintrust openai autoevals chromadb"
+    "%pip install -U /home/ubuntu/braintrust/braintrust/sdk/py openai autoevals chromadb"
```

Comment:
I am aware these must be changed back before merging. We just needed a couple small fixes to the SDK to make these examples work, so we need to push a new version of the SDK.